src/ripple/app/ledger/InboundLedger.h

```diff
@@ -99,6 +123,24 @@ class InboundLedger
     LedgerInfo
     deserializeHeader(Slice data, bool hasPrefix);
 
+    void
+    touch()
```

Comment:
I'd recommend having this accept a `clock_type::time_point` and eliminating the need to keep `m_clock` as a member. You'd also need to change the constructor to take a `clock_type::time_point` as well.

Reply:
I'm going to skip this one for now. I'm first trying to redesign the whole `InboundLedgers` + `InboundLedger` architecture.